### PR TITLE
V1.2.0

### PR DESCRIPTION
**Pull request by flyguy62n**
This PR implements three new features:

1. PDF file handler
2. CSV exports (to make auditor's jobs easier) 
3. Email data handler

It also closes a bug report that ended up being a really old version of Python on the user's end, but the net result is more robust handling when attempting to load the optional `wakepy` module (for sleep prevention).

Finally, it bumps the Windows embedded Python version to 3.12.10.

### CODE REVIEW

File: src/piidigger/datahandlers/email.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import re
+
+dhName='Primary Account Number'
```
Comment (rbclark) [inline src/piidigger/datahandlers/email.py]: This doesn't look right since it's repeated below

File: src/piidigger/datahandlers/email.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    '''
+    Matches a line of text against email address formats consistent with RFC5322.
+    Should receive the text from "filehandler" as raw text.
+
+    Returns a dictionary of:
+        'brand': set(matches)
+    '''
+    
+    # RFC5322 compliant email regex
+    regex=re.compile(r'(?:[a-zA-Z0-9!#$%&\'*+/=?^_`{|}~-]+(?:\.[a-zA-Z0-9!#$%&\'*+/=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?\.)+[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?|\[(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?|[a-zA-Z0-9-]*[a-zA-Z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])')
```
Comment (rbclark) [inline src/piidigger/datahandlers/email.py:15]: I'm getting a few false positives with this regex that seem like they would be filter-able, for example:

```
filename: C:\ProgramData\Microsoft\EdgeUpdate\Log\MicrosoftEdgeUpdate.log
matches:
    Email Address:
        email:
        - '&ap******************f@0.33'
        - r**@0.67
        - r**@0.33
        - '&ap*********************f@0.38'
        - r**@0.38
        - E77**********************************************************************************************************************************************************************************************************f@0.38
        - r**@0.75
        - '&ap*****************f@0.75'

filename: C:\ProgramData\Razer\Razer Central\Update\ProductUpdateModules.xml
matches:
    Email Address:
        email:
        - //a*********************************************************e@2x.png
        - //a*********************************************************r@2x.png
```
Would it be possible to improve the regex to be a little more restrictive?

Comment (rbclark) [reply src/piidigger/datahandlers/email.py:15]: On the other hand, it looks like this doesn't work right on international emails, using both the traditional and international examples from [wikipedia](https://en.wikipedia.org/wiki/International_email?useskin=vector) I'm only getting back 6 out of 14

```
filename: PIIDigger\test.txt
matches:
    Email Address:
        email:
        - '!#$****************~@example.com'
        - use****************************g@example.com
        - A*****3@example.com
        - '"Jo********"@example.com'
        - '"Fr**********"@example.com'
        - ste************5@gmail.com
```

**Review by rbclark — APPROVED** [review]
Looks good, I ran this locally and it found a lot of valid email addresses on my system, even in PDFs. I had a few questions regarding tuning the detection, but overall it currently already seems useful as is.

File: src/piidigger/datahandlers/email.py (hunk 2 above)
Comment (flyguy62n) [reply src/piidigger/datahandlers/email.py:15]: I've added additional validation logic and unit tests for the edge cases above.  On international, I'm not ready to talking i18n'ing PIIDigger, so I'm OK without it passing international addresses.

**Review by rbclark — APPROVED** [review]
Looks good to me, teted locally and found a lot of real emails and not a lot of false positives.
